src/Symfony/Bundle/FrameworkBundle/Command/TranslationUpdateCommand.php

```diff
@@ -144,8 +145,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
             ? new TargetOperation($currentCatalogue, $extractedCatalogue)
             : new MergeOperation($currentCatalogue, $extractedCatalogue);
 
+        $domains = $operation->getDomains();
+        $domain = $input->getOption('domain');
+
+        if (null !== $domain) {
+            $domains = array_intersect($domains, (array) $domain);
```

Comment:
I can update this. I don't know if passing multiple domain at once is really necessary.
Thanks for reviewing